src/Microsoft.AspNetCore.ResponseCompression/project.json

```diff
@@ -0,0 +1,34 @@
+{
+    "version": "1.1.0-*",
```

Comment:
Set this to `0.1.0-*` for the initial checkin. This will let us merge it sooner and iterate on it in the branch until it's ready for release.